dictionaries/software-terms/src/software-terms.txt

```diff
@@ -1704,3 +1704,4 @@ zip
 zlib
 ZPool
 ZPools
+trex
```

Comment:
@digger-yu,

What is `trex`?

Reply:
A tool from cisco

Reply:
@digger-yu,

Would you please add it to tools instead?

Reply:
@Jason3S  Revision completed